src/alltoallv.cu

```diff
@@ -0,0 +1,171 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <vector>
+#include "cuda_runtime.h"
+#include "common.h"
+
+int CHECK = 0;
+
+/**
+ * @brief Parses the parameter file and stores the matrix data into the imbalancingFactors reference passed in. 
+ * @param nranks The number of ranks in the test
+ * @param imbalancingFactors The reference to the vector that will store the parsed data
+ * @param filename The name of the parameter file to parse
+**/
+testResult_t parseParamFile(int nranks, std::vector<std::vector<double>> &imbalancingFactors, char filename[64]){
```

Comment:
filename[64] - you might want to use MAX_PATH or some other linux-defined thing rather than hardcoded length.

Reply:
`const` qualifier appropriate?